src/System/TimedScopes/TimedScopeStack.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using Microsoft.Omex.System.Logging;
+using Microsoft.Omex.System.Validation;
+
+namespace Microsoft.Omex.System.TimedScopes
+{
+	/// <summary>
+	/// LinkedStack
+	/// </summary>
+	[Serializable]
+	public class TimedScopeStack
+	{
+		/// <summary>
+		/// Root item for all stacks
+		/// </summary>
+		public static TimedScopeStack Root { get; } = new TimedScopeStack();
+
+
+		/// <summary>
+		/// Root node
```

Comment:
>/// Root node [](start = 2, length = 13)

The comment does not correspond to the property.

Shouldn't it be ReferenceEquals(this,Root)?

Reply:
I am not sure, it was done like this in OmexSystem code, Root method returns new TimedScopeStack().